src/app/components/Tasklist/Tasklist.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+type TasklistProps = {
+  activities: string[];
+  setActivities: (activities: string[]) => void;
+};
+
+export default function Tasklist({
+  activities,
+  setActivities,
+}: TasklistProps): JSX.Element {
+  const [newActivity, setNewActivity] = useState('');
+
+  function onClick(event: { preventDefault: () => void }) {
+    event.preventDefault();
+    const newActivities = [...activities, newActivity];
+    setActivities(newActivities);
+    setNewActivity('');
+  }
```

Comment:
This is all logic that should be outside this, on a page or in a separate component

Reply:
Thank you for the feedback. I will handle this next week in this issue
https://github.com/derjubu/capstone-project/issues/8